src/test/java/expediatestrunner/FilterTest.java

```diff
@@ -0,0 +1,30 @@
+package expediatestrunner;
+
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+import expediaproductobjects.FilterOptions;
+import expediaproductobjects.LandingPage;
+import expediaproductobjects.SearchBar;
+
+public class FilterTest extends Hooks {
+	
+	//this test searches for hotels with the given location and dates, and filters the results based on desired star rating and price range
+	@Test(enabled=true)
+	public void test() throws InterruptedException {
+		
+		SearchBar searchBar = new SearchBar(driver);
+		LandingPage landing = new LandingPage(driver);
+		FilterOptions filterOptions = new FilterOptions(driver);
+		
+		if(driver.findElement(By.xpath(landing.overlayCloseButton)).isDisplayed()) {
+			searchBar.click(landing.overlayCloseButton);
+		}
+		
+		searchBar.singleDestinationQuery("Boston, Massachusetts United States", "Wed Dec 01 2021", "Sat Dec 04 2021");
+		Thread.sleep(5000);
```

Comment:
Should use implicit wait

Reply:
Still dont understand the difference

Reply:
Implicit wait won't hard fail your code if it can't find the target element. If it can't find the target element it will wait n seconds. Thread.sleep will hard fail your code if the element is not found